tests/components/mill/test_init.py

```diff
@@ -109,7 +111,7 @@ async def test_unload_entry(hass):
     with patch.object(
         hass.config_entries, "async_forward_entry_unload", return_value=mock_coro(True)
     ) as unload_entry, patch("mill.Mill.connect", return_value=True):
-        assert await mill.async_setup_entry(hass, mock_entry)
+        assert await async_setup_component(hass, "mill", mock_entry)
 
         assert await mill.async_unload_entry(hass, mock_entry)
```

Comment:
Also update this on integration details:
```
assert await hass.config_entries.async_unload(entry.entry_id)
```

Reply:
Also there is a call to `mill.async_setup_entry` above that needs to be updated.